src/SFML/Window/OSX/SFContext.mm

```diff
@@ -31,9 +31,9 @@
 #include <SFML/Window/OSX/SFContext.hpp>
 #include <SFML/Window/OSX/WindowImplCocoa.hpp>
 
+#include <cstdint>
```

Comment:
Where's cstdint used here?

Reply:
It's replacing `stdint.h` as part of the `modernize-deprecated-headers check` check. I think it's just catching any uses of C headers when the C++ equivalent could be used instead.

Reply:
Yes, but that doesn't really answer the question 😄  
If we change a header and there's no change in code, is the header truly needed?

Reply:
Oh I see. I don't see anything in particular in this file that needs either header. I do know this header is already transitively included so we can definitely remove it from this file without breaking anything.